### PR TITLE
Russia rationalisation

### PR DESCRIPTION
**Pull request by si404**
This removes all subdivisions in Russia. Easily undoable, even at a later date, as the copies of the .csv files are what they are before this change.

It replaces 142 files with an average size of 2kB with 27 files with an average of 10.1kB. 3 routes are over 40kB and 2 more over 20kB. These big files might be troublesome for the server, but likewise the sheer number of pointless little routes that serve no purpose would have a similar effect.

Plan B, if this doesn't work, is to use https://en.wikipedia.org/wiki/Federal_districts_of_Russia - it reduces the number of routes, gets rid of nonsense like [this](http://cmap.m-plex.com/hb/hwymap.php?sys=eure&rg=all&gr=p&pg=5&r=ruslen.e20kra&showint=30&dl=1) and [this](http://cmap.m-plex.com/hb/hwymap.php?sys=eure&rg=all&gr=p&pg=6&r=rusmo.e30kar&showint=30&dl=1)  but breaks up the longer ones (but is more work for me).

15-08-21;Russia;E18;rus.e18;New Route
15-08-21;Russia;E20;rus.e20;New Route
15-08-21;Russia;E22;rus.e22;New Route
15-08-21;Russia;E28;rus.e28;New Route
15-08-21;Russia;E30;rus.e30;New Route
15-08-21;Russia;E30 (Chelyabinsk);rus.e30che;New Route
15-08-21;Russia;E38;rus.e38;New Route
15-08-21;Russia;E40;rus.e40;New Route
15-08-21;Russia;E50;rus.e50;New Route
15-08-21;Russia;E77 (Pskov);rus.e77;New Route
15-08-21;Russia;E77 (Kalingrad);rus.e77kal;New Route
15-08-21;Russia;E95;rus.e95;New Route
15-08-21;Russia;E97;rus.e97;New Route
15-08-21;Russia;E101;rus.e101;New Route
15-08-21;Russia;E105;rus.e105;New Route
15-08-21;Russia;E115;us.e115;New Route
15-08-21;Russia;E117;rus.e117;New Route
15-08-21;Russia;E119;rus.e119;New Route
15-08-21;Russia;E121;rus.e121;New Route
15-08-21;Russia;E123;rus.e123;New Route
15-08-21;Russia;E125;rus.e125;New Route
15-08-21;Russia;E127;rus.e127;New Route
15-08-21;Russia;E262;rus.e262;New Route
15-08-21;Russia;E391;rus.e391;New Route
15-08-21;Russia;E592;rus.e592;New Route
15-08-21;Russia;E017;rus.e017;New Route
15-08-21;Russia;All Routes;;Deleted all routes from all subdivisions

### CODE REVIEW

**Comment by jteresco** [comment]
I have no strong opinions about the change you made to eliminate subdivisions.  At least I have no problems with it.  I don't think file size will have any impact on the update process or the server - we still have around the same number of waypoints for these routes.  The place it will be seen is in highway browser and similar pages that have to load a lot of points.  I don't see that as a concern, though, as the newer mapping API handles lots of points amazingly well.  Also, it simplifies future drop-down menus that would have to list all those Russian subdivisions that I expect a very tiny fraction of users will ever visit or even browse much.

I won't pull this one in until some of the other experienced collaborators have had a chance to weigh in.

**Comment by si404** [comment]
Thanks Jim - my main concerns were technical.

It does break .list files of ua747 and 1995hoo from CHM, but the former's routes are hard to follow and the latter has a tiny bit of one road.

By all means wait a while, before pulling, I didn't expect otherwise!

**Comment by jteresco** [comment]
Also, no need to have the copies of the csv files in the repository separately.  GitHub is maintaining a complete history for us.  If you go to any file, you can open it up, click on History, and see every version that's ever been committed.

**Comment by yakra** [comment]
Not looking into it too deeply just yet....
How does it break the list files?

**Comment by si404** [comment]
Eric - they've clinched routes in Russia, but as I'm removing the subsystems, they will need to change their .list files to have them clinched.

**Comment by jteresco** [comment]
Are there any further updates entries needed for the additional commits that are now part of this pull request?  If I don't heard objections, I'll pull this in soon, likely Tuesday evening.

**Comment by si404** [comment]
15-08-25;Italy;A1 (Sasso Marconi);ita.a001sas;New Route
15-08-25;Italy;A4 (Chivasso);ita.a004chi;Route extended at north end from A4 to SP81 and SP91
15-08-25;Italy;A6;ita.a006;Route extended at north end from A55 to Corso Trieste
15-08-25;Italy;A6 (Fossano);ita.a006fos;New Route
15-08-25;Italy;A11 (Viareggio);ita.a011via;New Route
15-08-25;Italy;A11A12;;Route renamed A11 (Viareggio)
15-08-25;Italy;A13 (Ferrara);ita.a013fer;Route extended at east end from SS16 to SS309
15-08-25;Italy;A14 (Ravenna);;Route renamed A14dir (Ravenna)
15-08-25;Italy;A14 (Bari);ita.a014bar;New route
15-08-25;Italy;A14dir (Ravenna);ita.a014dirrav;New Route
15-08-25;Italy;A18 (Catania);;Route renamed A18dir (Catania)
15-08-25;Italy;A18dir (Catania);ita.a018dircar;New Route
15-08-25;Italy;A19 (Palermo);;Route merged into A19
15-08-25;Italy;A19;ita.a019;Route extended at west end from SS121 to Via Oreto
15-08-25;Italy;A27;ita.a027;Route extended at south end from A57 to SS14
15-08-25;Italy;A29;ita.a029;Route extended at east end from Via Rosario Nicoletti to southern junction with Viale Regione Siciliana Nord Ovest
15-08-25;Italy;A29 (Palermo);;Route merged into A29
15-08-25;Italy;A29 (Trapani);;Route renamed A29Dir (Trapani)
15-08-25;Italy;A29 (Birgi);;Route renamed A29Dir (Birgi)
15-08-25;Italy;A29dir (Birgi);ita.a029dirbir;New Route
15-08-25;Italy;A29dir (Trapani);ita.a029dirtra;New Route
15-08-25;Italy;A31;ita.a031;Route extended at south end from SP8 to SP434
15-08-25;Italy;A35;ita.a035;New Route
15-08-25;Italy;A36;ita.a036;New Route
15-08-25;Italy;A52 (Rho);ita.a052rho;New Route
15-08-25;Italy;A55;ita.a055;Route truncated at west end from junction with SS25 to junction with Rivoli spur of the A55
15-08-25;Italy;A55;ita.a055;Route extended at west end from junction with Rivoli spur to to junction with SS29
15-08-25;Italy;A55 (Rivoli);ita.a055riv;New Route
15-08-25;Italy;A55 (Moncalieri);;Route merged into A6
15-08-25;Italy;A55 ((South) Torino);;Route merged into main A55
15-08-25;Italy;A57 (Venezia);;Route merged into A27
15-08-25;Italy;A58;ita.a058;New Route
15-08-25;Italy;A59;ita.a059;New Route
15-08-25;Italy;A60;ita.a060;New Route
15-08-25;Italy;A91;ita.a091;Route extended at west end from Via Renato Ciprari to Via Alessandro Guidoni
15-08-25;Italy;Autostrade Sistiana-Rabuiese;autsisrab;New Route
15-08-25;Italy;E31;ita.e31;Route extended at south end from A12 to Via Bologna
15-08-25;Italy;E45 (Roma);ita.e45rom;New Route
15-08-25;Italy;E61;ita.e61;Route removed from SR58 and SR14 and relocated onto RA14 and RA13 between SR58 and SR14
15-08-25;Italy;E64;ita.e64;Route truncated at eastern end from Corso Vercelli to E70
15-08-25;Italy;E74;ita.e74;Route removed from SS231 and relocated onto A33 between SS231_N and SS231_S
15-08-25;Italy;E612;ita.e612;Route truncated at southern end from A4/A55 junction to A55 junction (now called A5_S)
15-08-25;Italy;E612;ita.e612;Route extended at southern end from southern A5 junction to A55 junction
15-08-25;Italy;E751;ita.e751;New Route
15-08-25;Italy;Dramazione per Fernetti;;Route renamed RA14
15-08-25;Italy;RA1;ita.ra001;New Route
15-08-25;Italy;RA1 (Borgo Panigale);ita.ra001bor;New Route
15-08-25;Italy;RA1;ita.ra002;New Route
15-08-25;Italy;RA1;ita.ra003;New Route
15-08-25;Italy;RA1;ita.ra005;New Route
15-08-25;Italy;RA1;ita.ra009;New Route
15-08-25;Italy;RA1;ita.ra011;New Route
15-08-25;Italy;RA1;ita.ra012;New Route
15-08-25;Italy;RA1;ita.ra013;New Route
15-08-25;Italy;RA1;ita.ra014;New Route
15-08-25;Italy;RA1;ita.ra015;New Route
15-08-25;Italy;RA1;ita.ra016;New Route
15-08-25;Italy;R.A. Ascoli-Porto d'Ascoli;;Route renamed RA11
15-08-25;Italy;R.A. di Benevento;;Route renamed RA9
15-08-25;Italy;R.A. Chieti-Pescara;;Route renamed RA12
15-08-25;Italy;R.A. Ferrara-Porto Garibaldi;;Route merged with A13 (Ferrara)
15-08-25;Italy;R.A. Pordenone; renamed RA16
15-08-25;Italy;R.A. Salerno-Avellino;;Route renamed RA2
15-08-25;Italy;R.A. Scalo Sicignano-Potenza;;Route renamed RA5
15-08-25;Italy;R.A. Siena-Firenze;;Route renamed RA3
15-08-25;Italy;R.A. Torino-Aeroporto di Caselle;;Route renamed RA10
15-08-25;Italy;R.A. Trieste;;Route renamed RA13
15-08-25;Italy;Tangenziale di Bologna;;Route renamed RA1
15-08-25;Italy;Tangenziale di Bologna (Borgo Panigale);;Route renamed RA1 (Borgo Panigale)
15-08-25;Italy;Tangenziale di Catania; renamed RA15
15-08-25;Italy;T1;ita.t001;New Route
15-08-25;Italy;T2;ita.t002;New Route
15-08-25;Italy;T4;ita.t004;New Route

**Comment by si404** [comment]
15-08-25;Spain;A2 (Girona);esp.a002gir;Route extended at north end from exit 708 to junction with the AP7
15-08-25;Spain;A2 (Girona);esp.a002gir;Route extended at south end from exit 700 to junction with the C63
15-08-25;Spain;AP2;esp.ap002;Route extended west from western junction with AP7 to exit 10 of the B23
15-08-25;Spain;AP2 (Molins de Rei);;Route deleted
15-08-25;Spain;AP4;esp.ap004;Route extended at north end from exit 15 to exit 13
15-08-25;Spain;A7;esp.a007;Route extended at south end from exit 375 to exit 355: includes Castell de Ferro section
15-08-25;Spain;A7 (Castell de Ferro);;Route deleted: merged into main A7
15-08-25;Spain;A7 (Malaga);esp.a007mal;Route extended east from exit 314 to exit 342: includes Motril section
15-08-25;Spain;A7 (Motril);;Route deleted: merged into Malaga section
15-08-25;Spain;A8;esp.a008;Route extended at west end from exit 199 to exit 202
15-08-25;Spain;A8 (Gijon);esp.a008gij;extended e from 291->228 includes Cabezón de la Sal section
15-08-25;Spain;A8 (Gijon);esp.a008gij;extended w from 536->593 includes Vilalba section
15-08-25;Spain;A8 (Cabezón de la Sal);;Route deleted: merged into Gijon section
15-08-25;Spain;A8 (Vilalba);;Route deleted: merged into Gijon section
15-08-25;Spain;AP9 (Ferrol);esp.ap009fer;truncated at north end from Paseo de Marina to exit 36F
15-08-25;Spain;A11 (Aranda de Duero);esp.a011ara;New Route 
15-08-25;Spain;A12;esp.a012;Route extended at west end from exit 76 to exit 78
15-08-25;Spain;A12 (Santo Domingo de la Calzada);;Route deleted
15-08-25;Spain;A12 (Nájera);esp.a012naj;Route extended at west end from exit 28 to exit 44
15-08-25;Spain;A15;esp.a015;Route extended at north end from exit 164 to AP1/8
15-08-25;Spain;A15 (Astigarraga);;Route merged with main A15
15-08-25;Spain;A15 (Pamplona);;Route merged into AP15
15-08-25;Spain;A15 (Soria);esp.a015sor;;Reentered Route
15-08-25;Spain;AP15;esp.ap015;Route extended at north end from exit 83A to exit 112
15-08-25;Spain;AP15 (Irurtzun);;Route merged with main AP15
15-08-25;Spain;A23 (Sabinanigo);esp.a023sab;New Route
15-08-25;Spain;A32 (Albacete);esp.a032alb;New Route
15-08-25;Spain;A38 (Gandia);esp.a038gan;New Route
15-08-25;Spain;A54 (Lugo);esp.a054lug;New Route
15-08-25;Spain;A66 (León);;Route merged into main A66
15-08-25;Spain;A66;esp.a066;Route extended at north end from exit 268 to exit 143
15-08-25;Spain;A77a;;Route deleted
15-08-25;Spain;A318;esp.a318;New Route
15-08-25;Spain;A376;;Route deleted
15-08-25;Spain;A382;;Route deleted
15-08-25;Spain;A480;;Route deleted
15-08-25;Spain;A497;;Route deleted
15-08-25;Spain;A601;esp.a601;Route extended at north end from exit 36 to exit 4
15-08-25;Spain;AC14;esp.ac014;Route extended at north end from exit 3 to exit 1
15-08-25;Spain;B22 (Terminal 1);esp.b022ter;New Route
15-08-25;Spain;B40 (La Roca de Valles);esp.b040roc;New Route
15-08-25;Spain;BA20;;Route deleted
15-08-25;Spain;BI631;esp.bi631;New Route
15-08-25;Spain;BI636;esp.bi636;New Route
15-08-25;Spain;BI644;;Route deleted
15-08-25;Spain;BU30;esp.bu030;Route extended at east end from exit 23 to exit 19
15-08-25;Spain;C13;esp.c013;New Route
15-08-25;Spain;C14;esp.c014;New Route
15-08-25;Spain;C15;esp.c015;New Route
15-08-25;Spain;C16;esp.c016;Route truncated at north end from N260 to exit 95
15-08-25;Spain;C17;esp.c017;Route truncated at south end from B20 to exit 44
15-08-25;Spain;C17;esp.c017;Route truncated at north end N260 to exit 91
15-08-25;Spain;C31 (el Prat de Llobregat);esp.c031pra;New Route 
15-08-25;Spain;C31 (Palamos);esp.c031pal;New Route
15-08-25;Spain;C35;esp.c035;New Route
15-08-25;Spain;C58;esp.c058;Route truncated at west end from C-55 to C-16
15-08-25;Spain;C58C;esp.c058c;New Route
15-08-25;Spain;C65;esp.c065;New Route
15-08-25;Spain;CM10;esp.cm010;New Route
15-08-25;Spain;CO32;esp.co032;New Route
15-08-25;Spain;CV10;esp.cv010;Route truncated at north end from N232 to exit 41 and extended at north end from exit 41 to exit 45.
15-08-25;Spain;CV35;esp.cv035;New Route
15-08-25;Spain;CV365;;Route deleted
15-08-25;Spain;E1 (Ferrol);esp.e1fer;Route truncated at north end from Paseo de Marina to exit 36F
15-08-25;Spain;E1 (Vigo);esp.e1vig;New Route
15-08-25;Spain;E5;esp.e5;Removed from M40 and relocated onto A1, M30 and A4 between A1 exit 13 and A4 exit 6
15-08-25;Spain;E5 (Bilbao);esp.e5bil;New Route
15-08-25;Spain;E5 (Beasain);esp.e5bea;New Route
15-08-25;Spain;E15 (Cartagena);esp.e15car;New Route
15-08-25;Spain;E901;esp.e901;Route extended at west end from A3 exit 6 to A3 exit 3
15-08-25;Spain;EL20;esp.el020;Route extended south end from 3 to 4
15-08-25;Spain;EX-A1;esp.exa001;Route extended at west end from exit 82 to exit 99
15-08-25;Spain;EX-A1 (Moraleja);;Route merged with main EX-A1
15-08-25;Spain;FE15;;Route deleted
15-08-25;Spain;FV2;esp.fv002;New Route
15-08-25;Spain;GC1;esp.gc001;Route extended south end from exit 62 to exit 68
15-08-25;Spain;GC2;esp.gc002;New Route
15-08-25;Spain;GC2 (Galdar);esp.gc002gal;New Route
15-08-25;Spain;GI632;esp.gi632;New Route
15-08-25;Spain;GR16;esp.gr016;New Route
15-08-25;Spain;GR30;esp.gr030;New Route
15-08-25;Spain;LU11;;Route deleted
15-08-25;Spain;M30 (Bypass Sur);esp.m030bypsur;New Route
15-08-25;Spain;MP203;esp.mp203;New Route
15-08-25;Spain;M407;esp.m407;New Route
15-08-25;Spain;M501;esp.m501;New Route
15-08-25;Spain;M503;esp.m503;New Route
15-08-25;Spain;M607;esp.m607;New Route
15-08-25;Spain;MA22;;Route deleted
15-08-25;Spain;MA23;esp.ma023;New Route
15-08-25;Spain;PO12;;Route deleted
15-08-25;Spain;RM11;esp.rm011;New Route
15-08-25;Spain;RM16;esp.rm016;New Route
15-08-25;Spain;RM17;esp.rm017;New Route
15-08-25;Spain;S30;esp.s030;Route extended at south end from exit 5 to S-10
15-08-25;Spain;SA11;;Route deleted
15-08-25;Spain;SA20;;Route deleted
15-08-25;Spain;SC11;;esp.sc011;Route truncated at north end from SC20 to Rua da Chaparra
15-08-25;Spain;SC12;;Route deleted
15-08-25;Spain;SG20;;Route deleted
15-08-25;Spain;SO20;esp.so020;Route truncated at north end from exit 21 to exit 9
15-08-25;Spain;TF5;esp.tf005;Route truncated at west end from TF-42 to exit 39
15-08-25;Spain;TO21;esp.to021;Route extended at south end from CM-40 to the southern junction with N-403
15-08-25;Spain;V15;;Route deleted
15-08-25;Spain;VA20;;Route deleted
15-08-25;Spain;VA30;esp.va030;Route extended at east end from exit 13 to exit 
15-08-25;Spain;Z32;esp.z032;New Route

**Comment by jteresco** [comment]
Si, I see you pulled in 59, so if you want to pull this in too, that's ok with me.  I'll copy in the systems.csv entries and try an update with all of this tonight.

**Comment by jteresco** [comment]
Even better, you already have the systems.csv in there.  Trying an update now.

**Comment by jteresco** [comment]
Fixing typos in the csv files, and also encodings.  We had a mix of various encodings.  To keep my python program (and any other programs that might read the raw data we're collecting) happy, I'd like to make sure all files are plain ascii when no international characters are used, and UTF-8 with no byte-order markers when they are.  When I commit tonight's edits to get the csv changes back to the master, this will again be the case.

**Comment by jteresco** [comment]
Errors I can't handle with simple edits: 

hwy_data/ESP/espa/esp.ap002mol.wpt is missing.  I'm deleting its csv entries for now.
hwy_data/ESP/espa/esp.a012san.wpt missing, also deleting csv entries.
hwy_data/WY/usawy/wy.wy014.wpt missing, also deleting its csv entries.

**Comment by jteresco** [comment]
esp.e5.wpt had the following 3 lines in succession:

1(AP1) http://www.openstreetmap.org/?lat=42.334755&lon=-3.622634
AP1/A1_S http://www.openstreetmap.org/?lat=42.334755&lon=-3.622634
242(A1) http://www.openstreetmap.org/?lat=42.334755&lon=-3.622634

Since all are the same point, I assume these are alternate names for the same point.  If not, please fix after I commit these changes.

Same error for esp.e80.wpt:

4(BU30) http://www.openstreetmap.org/?lat=42.303595&lon=-3.754191
4 +BU30 http://www.openstreetmap.org/?lat=42.303595&lon=-3.754191

**Comment by jteresco** [comment]
Another one: esp.c031pra.wpt is an empty file.  Deleting its csv entries pending fix.
